packages/view-types/feature-list/src/FeatureList.js

```diff
@@ -66,6 +69,14 @@ export default function FeatureList(props) {
     setSearchTerm(event.target.value);
   };
 
+  let columns = ['name'];
+  let columnLabels = [`${capitalize(featureType)} ID`];
+
+  if (showTable) {
+    columns.push('key');
+    columnLabels.push('Alternate ID');
+  }
```

Comment:
@keller-mark in here I am hardcoding `name` and `key`. `name` was already hardcoded, so I assume that we are sure that it will always be present in `data`. What about `key`? Can we be sure that `data` will always contain `key`? I don't want this table to start failing for some datasets. Also, is there any other part of data, apart from `key` and `name` that users might be interested in and that we should add handling for?

Reply:
`data` is generated locally in `<FeatureList/>` so we have full control over it - it always has `key` and `name` https://github.com/vitessce/vitessce/blob/4384c5abcb6ad7f1f231165e7e5c10de98ff6dc5/packages/view-types/feature-list/src/FeatureList.js#L58